### PR TITLE
Update README for legacy branch

### PR DESCRIPTION
**Pull request by heaths**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Azure service crates generated from [Azure REST API Specifications](https://gith
 
 ## Status
 
-🚨 WARNING 🚨: This project is under active development. Be aware that large breaking changes will happen before 1.0 is reached.
+🚨 WARNING 🚨: This project is no longer under active development. Work on officially supported crates is being done in the `main` branch. Pull requests to `legacy` may be considered but are not guaranteed to be merged. Instead, you might consider forking this repository, creating a topic branch you'd need to submit a pull request anyway, and [override dependencies](https://doc.rust-lang.org/cargo/reference/overriding-dependencies.html) in your `Cargo.toml` file(s) as necessary.
```
Comment (RickWinter) [inline README.md:33]: I don't like suggesting they fork. I'd rather they migrate to the new crates

Comment (heaths) [reply README.md:33]: That text is already above. I'm taking into account - much further below in this file - that they may not be *able* to, especially before they are ready. We could opt not to suggest something (and I intentionally do not go into too many details), but that may just lead to more issues being opened. If that's the case, we should consider a bot response.

Comment (RickWinter) [reply README.md:33]: Can we say please migrate to the new crates or open issues/PRs against main. Forking as a last resort .....

// @ronniegeraghty Any opinions on this wording?

Comment (RickWinter) [reply README.md:33]: > That text is already above. I'm taking into account - much further below in this file - that they may not be _able_ to, especially before they are ready. We could opt not to suggest something (and I intentionally do not go into too many details), but that may just lead to more issues being opened. If that's the case, we should consider a bot response.

Makes sense. I'm hoping to get issues so we learn who was using these crates and which ones we might want to prioritize

**Review by RickWinter — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Azure SDK for Rust
 
-This repository is for the development of the [unofficial](https://github.com/Azure/azure-sdk-for-rust/blob/main/FAQ.md#why-is-it-unofficial) Azure SDK for Rust.
+This repository is for the development of the [unofficial, unsupported](https://github.com/Azure/azure-sdk-for-rust/blob/legacy/FAQ.md#why-is-it-unofficial) Azure SDK for Rust.
 
 > Microsoft is developing the official Azure SDK for Rust crates and has no plans to update this unofficial crate.
```
Comment (ronniegeraghty) [inline README.md:5]:
```suggestion
> Microsoft is developing the official Azure SDK for Rust crates and has no plans to update these unofficial crates.
```

**Comment by ronniegeraghty** [comment]
Is this the text for the individual track 1 package `README.md` files? Or is this for the main `README.md` in the legacy branch? 
Just some grammar to fix if it's for the main `README.md` to reference packages and not a package.

**Comment by heaths** [comment]
> Is this the text for the individual track 1 package `README.md` files? Or is this for the main `README.md` in the legacy branch? Just some grammar to fix if it's for the main `README.md` to reference packages and not a package.

It's the `README.md` in the root, which you can see in the Files tab.

File: README.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 # Azure SDK for Rust
 
-This repository is for the development of the [unofficial](https://github.com/Azure/azure-sdk-for-rust/blob/main/FAQ.md#why-is-it-unofficial) Azure SDK for Rust.
+This repository is for the development of the [unofficial, unsupported](https://github.com/Azure/azure-sdk-for-rust/blob/legacy/FAQ.md#why-is-it-unofficial) Azure SDK for Rust.
 
-> Microsoft is developing the official Azure SDK for Rust crates and has no plans to update this unofficial crate.
+> Microsoft is developing the official Azure SDK for Rust crates and has no plans to update these unofficial crates.
 > In the future we may release an official version that may have a different package name.
```
Comment (ronniegeraghty) [inline README.md:6]:
```suggestion
> In the future we may release official versions that may have a different package names.
```

File: README.md (hunk 1 above)
Comment (ronniegeraghty) [reply README.md:33]: Maybe adding something like: "These crates should NEVER be used in a production setting."

Comment (heaths) [reply README.md:33]: Too late. But at least for 1P, the "unsupported" should've implied that.

**Review by ronniegeraghty — APPROVED** [review]

**Pull request merged**